2sem/common/stack.h

```diff
@@ -0,0 +1,100 @@
+﻿#pragma once
+
+/// <summary>
+/// Класс Stack реализует стек с использованием связного списка.
+/// Стек работает по принципу "последний пришел — первый вышел" (LIFO).
+/// </summary>
+template <typename T>
+class Stack
+{
+private:
+    /// <summary>
+    /// Внутренняя структура, представляющая узел стека.
+    /// </summary>
+    struct Node
+    {
+        T data; //< Данные, хранящиеся в узле.
+        Node* next; //< Указатель на следующий узел в стеке.
+    };
+
+    Node* top = nullptr; //< Указатель на вершину стека.
```

Comment:
Вы можете нажать кнопку Applay suggestion и предложенные изменения применятся. А так вы написали, что исправили, хотя видимо пропустили
```suggestion
    Node* top = nullptr; // Указатель на вершину стека.
```

Reply:
Исправлено